src/utils/metadata.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+import { getSetting } from 'docc-render/utils/theme-settings';
+import { absoluteURL } from 'docc-render/utils/assets';
+
+const themeTitle = getSetting(['meta', 'title'], process.env.VUE_APP_TITLE);
+const DEFAULT_DESCRIPTION = 'There\'s never been a better time to develop for Apple platforms.';
+const createMetaTags = ({ title, description, path }) => [
+  {
+    name: 'description',
+    content: description,
+  },
+  {
+    property: 'og:locale',
+    content: 'en_US',
+  },
+  {
+    property: 'og:site_name',
+    content: themeTitle,
+  },
+  {
+    property: 'og:type',
+    content: 'website',
+  },
+  {
+    property: 'og:image',
+    content: absoluteURL('/developer-og.jpg'),
+  },
+  {
+    property: 'og:title',
+    content: title,
+  },
+  {
+    property: 'og:description',
+    content: description,
+  },
+  {
+    property: 'og:url',
+    content: absoluteURL(path),
+  },
+  {
+    name: 'twitter:card',
+    content: 'summary_large_image',
+  },
+  {
+    name: 'twitter:image',
+    content: absoluteURL('/developer-og-twitter.jpg'),
```

Comment:
Is the twitter-specific one just a different padding and/or aspect ratio? Just curious.

Reply:
Different size and different ratio, yes. That's why they are different pictures.